challenge_4/go/erocs/invert_tree.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+)
+
+type TreeNode struct {
+	Left  *TreeNode
+	Right *TreeNode
+	Value int
+}
+
+func (t *TreeNode) String() string {
+	l := ""
```

Comment:
Either works fine. I find explicit initialization to be more descriptive but each to his own.